testdata/echo.EchoService.Hello.jsonnet

```diff
@@ -0,0 +1,5 @@
+function(input) {
+  response: {
+    response: 'Hello ' + input.request.message,
```

Comment:
Needs a pony imo. happy to PR it ;)
also, as mentioned before, would change message and response to e.g.
first_name and greeting.